tests/jepsen/run.sh

```diff
@@ -184,6 +184,37 @@ PROCESS_RESULTS() {
     INFO "Result processing (printing and packing) DONE."
 }
 
+CLUSTER_UP() {
+  PRINT_CONTEXT
+  "$script_dir/jepsen/docker/bin/up" --daemon
+  sleep 1
+  # Ensure all SSH connections between Jepsen containers work
+  for node in $(docker ps --filter name=jepsen* --filter status=running --format "{{.Names}}"); do
+      if [ "$node" == "jepsen-control" ]; then
+          continue
+      fi
+      node_hostname="${node##jepsen-}"
+      docker exec jepsen-control bash -c "ssh -oStrictHostKeyChecking=no -t $node_hostname exit"
```

Comment:
Is 1 sec sleep deterministic enough to be confident that it is up?

I don't want the situation where we have a healthy cluster that we just brought down just to then bring it up, but didn't wait long enough for it to respond to the ssh check.

Reply:
Your consern makes sense, I should push it a bit longer, what do you think 5s or 10s?

Reply:
Could we make it time independent?
Perhaps a loop where we retry for a couple of times and fail after a certain number of tries?

Reply:
I understand, but I would like to merge ASAP because there is operational cost to making this PR wait for merge. This loop checking is a good idea, but it may be an overkill at the moment. Let's merge and if problems start popping up, which I don't think will, I will make the change. What do you think?